src/components/data-management/DataManagementIntercept.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import {
+  Modal, Typography, Space, Button,
+} from 'antd';
+import { runGem2s } from 'redux/actions/pipeline';
+import { useRouter } from 'next/router';
+
+const { Text } = Typography;
+
+const DataManagementIntercept = (props) => {
+  const {
+    onContinueNavigation, onDismissIntercept, rerunStatus, experimentId,
+  } = props;
+  const dispatch = useDispatch();
+  const router = useRouter();
+  return (
+    <Modal
+      title='Changes not applied'
+      visible
```

Comment:
Clicking on the `x` close button on the top right part of the modal doesn't dismiss it. Add a handler for that event by adding a callback to `onCancel` on the modal.